www/src/pages/components/form-field.mdx

```diff
@@ -0,0 +1,202 @@
+    </>
+  );
+}
+```
+
+
+##### Input types
+
+```jsx live
+() => {
+  const [value, setValue] = useState('');
+  return (
+    <>
+      <FormField size="sm">
+        <FormControlDecoratorGroup floatingLabel="What kind of cats?">
+          <FormFieldControl
+            leadingElement={<Icon src={FavoriteBorder} />}
+            value={value}
+            onChange={(e) => setValue(e.target.value)}
+          />
+        </FormControlDecoratorGroup>
+      </FormField>
+      <FormField>
+        <FormControlDecoratorGroup floatingLabel="What kind of cats?">
+          <FormFieldControl
+            as="textarea"
+            leadingElement={<Icon src={FavoriteBorder} />}
+            value={value}
+            onChange={(e) => setValue(e.target.value)}
+          />
+        </FormControlDecoratorGroup>
+      </FormField>
+      <FormField size="lg">
+        <FormControlDecoratorGroup floatingLabel="What kind of cats?">
+          <FormFieldControl
+            as="select"
+            leadingElement={<Icon src={FavoriteBorder} />}
+            value={value}
+            onChange={(e) => setValue(e.target.value)}
+          >
+            <option value="">Select a cat</option>
+            <option>Persian cat</option>
+            <option>British Shorthair</option>
+            <option>Maine Coon</option>
+            <option>Siamese cat</option>
+          </FormFieldControl>
+        </FormControlDecoratorGroup>
+      </FormField>
+    </>
+  );
+}
+```
+
+
+
+<PropsTable {...props.data.formFieldMetadata} />
+<PropsTable {...props.data.formFieldLabelMetadata} />
+<PropsTable {...props.data.formFieldControlMetadata} />
+<PropsTable {...props.data.formControlDecoratorGroupMetadata} />
+
+export const query = graphql`
+  query {
+    formFieldMetadata: componentMetadata(displayName: { eq: "FormField" }) {
+      ...ComponentDocGenData
+    }
+    formFieldLabelMetadata: componentMetadata(displayName: { eq: "FormFieldLabel" }) {
+      ...ComponentDocGenData
+    }
+    formFieldControlMetadata: componentMetadata(displayName: { eq: "FormFieldControl" }) {
+      ...ComponentDocGenData
+    }
+    formControlDecoratorGroupMetadata: componentMetadata(displayName: { eq: "FormControlDecoratorGroup" }) {
+      ...ComponentDocGenData
+    }
+  }
+`
+
+
+```
+
+<TextField
+      className="mb-3"
+      leadingElement={<Icon src={Add} />}
+      trailingElement={<Icon src={Add} />}
+      label="Last Name"
+    />
+
+    <TextField
+      as="textarea"
+      className="mb-3"
+      leadingElement={<Icon src={Add} />}
+      trailingElement={<Button size="sm" variant="tertiary">Copy</Button>}
+      label="Last Name"
+    />
+
+    <SelectField
+      className="mb-3"
+      leadingElement={<Icon src={Add} />}
+      trailingElement={<Button>Copy</Button>}
+      label="Last Name"
+    >
+      <option>1</option>
+      <option>2</option>
+      <option>3</option>
+      <option>4</option>
+      <option>5</option>
+    </SelectField>
+```
+
+
```

Comment:
If it has different states for success or error, I'd love to see those in the docs. :)

Reply:
For sure :) When I add those let's make sure those examples are there.